### ISSUE TITLE
statx: Operation not permitted

### ISSUE THREAD

**Issue by ptm108** [neo4j/docker-neo4j#339]
Hi, facing an error w statx while trying to add SSL support to an existing neo4j project. 

Container logs:
```
stat: cannot statx '/ssl': Operation not permitted

Folder /ssl is not accessible for user: 1003 or group 1003 or groups 1003, this is commonly a file permissions issue on the mounted folder.

Hints to solve the issue:
1) Make sure the folder exists before mounting it. Docker will create the folder using root permissions before starting the Neo4j container. The root permissions disallow Neo4j from writing to the mounted folder.
2) Pass the folder owner's user ID and group ID to docker run, so that docker runs as that user.
If the folder is owned by the current user, this can be done by adding this flag to your docker run command:
  --user=$(id -u):$(id -g)
```

I've tried all 3 hints but none has worked so far

Docker version: 18.03.1
Neo4j version: 4.4.4

**Comment by jennyowen** [comment]
OK what are the file permissions on the folder you're mounting to `/ssl`? 
What about the files inside that folder, are they also readable by the user you're running docker as?

**Comment by ptm108** [comment]
Thanks for the quick reply, I've tried running a `chmod -R 777` on the ssl/ folder. Also changed the folder owner to the user. None of these worked for me

**Comment by jennyowen** [comment]
This is a duplicate of #318. I'm going to close this issue and continue it over on #318.
